docs/source/basic_tutorials/non_core_models.md

```diff
@@ -0,0 +1,15 @@
+# Non-core Model Serving
+
+TGI supports various LLM architectures (see full list [here](./supported_models)). If you wish to serve a model that is not one of the supported models, TGI will fallback to transformers implementation of that model. This means you will be unable to use some of the features introduced by TGI, such as tensor-parallel sharding or flash attention. However, you can still get many benefits of TGI, such as continuous batching or streaming outputs.
+
+You can serve these models using Docker like below 👇 
+
+```bash
+docker run --gpus all --shm-size 1g -p 8080:80 -v $volume:/data ghcr.io/huggingface/text-generation-inference:latest --model-id gpt2
```

Comment:
Is there any difference in the command  with normal docker run for supported models?

Reply:
From what I get there's two type of unsupported models. One is transformers models without custom kernels, and the other one is non-transformer models. For non-transformer models since they have the model implementation inside repo you just need to pass --trust-remote-code and that's it. For transformers models since you don't have any flash attention or rope scaling or additional good features of TGI you don't need to pass anything. (Above code is what I run on my local so I can confirm)

Reply:
Yes, but this appears to be the same as the example in the Quick tour: https://moon-ci-docs.huggingface.co/docs/text-generation-inference/pr_906/en/quicktour. I'd maybe clarify it so users are more confident on what they are doing.

Reply:
@pcuenca I think it's due to how you still run them the same way, except you can't run with custom flags that is not available for one model, that's why I still added the same command.

Reply:
I committed your suggestion to clarify this.